site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -200,7 +200,7 @@ const CreateConsequenceNetwork = (props: CreateConsequenceNetworkProps): ReactEl
                             returnPath={returnPath}
                         />
 
-                        {(props.consequenceIndex || 0) <= 10 && (
+                        {(props.consequenceCount || 0) < MAX_CONSEQUENCES && (
```

Comment:
Logic needs modifying slightly. When creating a 15th consequence the "add another consequence" button shows because the 15th disruption isn't in the database yet. This means you can go through the flow to create a 16th consequence - although it will eventually fail on the api check